Documentation/articles/getting_started/1_creating_a_new_project_vs.md

```diff
@@ -1,19 +1,75 @@
 # Creating a Project with Visual Studio
 
-Start up Visual Studio and select **New Project...** in the upper left corner.
+MonoGame is available on Windows for development using the instructions below to get started with MonoGame 3.8.
+
+> Note, MonoGame 3.8 projects are no longer compatible with earlier versions of MonoGame.  If you wish to work on or build older MonoGame projects, then you will still need to install [MonoGame 3.7.1](https://www.monogame.net/downloads/) or earlier to open them.
+
+## Prerequisites
+
+You will need a copy of [Visual Studio 2019](https://www.monogame.net/downloads/) or later installed (any edition, including Community) before installing MonoGame, with the following components (depending on your target platform):
```

Comment:
I would get rid of the 'or later' part. We'll need to make sure things work with the next version of Visual Studio and we should update our docs when we do that.

Reply:
Prereq's moving to the development req's page instead.